### DISCUSSION TITLE
Date Picker

### DISCUSSION THREAD

**Discussion by abrorxon-karimxonov** [mantinedev/mantine#4510] [Q&A]
### What package has an issue

@mantine/dates

### Describe the bug

hi there  can you help me?. how can i control next and prev month date pickers pls help me

### What version of @mantine/hooks page do you have in package.json?

6.0.16

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by abrorxon-karimxonov** [comment]
i have two btn next and prev i want to after every clecked should be changed month

**Comment by rtivital** [comment]
Please use GitHub issues only to report issues, to ask questions use GitHub discussions.

To control date, see documentation – https://mantine.dev/dates/date-picker/#controlled-date

**Comment by abrorxon-karimxonov** [comment]
thank you
